test/wdio/global.ts

```diff
@@ -0,0 +1,9 @@
+import { setMode } from '@stencil/core';
+
+const globalScript = () => {
+  setMode((elm) => {
+    return (elm as any).colormode || elm.getAttribute('colormode') || (window as any).KarmaMode;
```

Comment:
```suggestion
    return (elm as any).colormode || elm.getAttribute('colormode');
```

Reply:
ah makes sense :)

Reply:
gonna be honest, I moved this file without reading it that closely :O

Reply:
haha .. been doing that too 🙈